tsconfig.json

```diff
@@ -8,7 +8,6 @@
     "./*.js",
     "./**/jest.config.ts",
     "./ci/**/*.ts",
-    "./resources/notarize.ts",
```

Comment:
Out of curiosity: this was removed cause it was no longer needed? Why?

Reply:
Yes, because the file no longer exists.